app/main.py

```diff
@@ -4,0 +1,74 @@
+import json
+from os import path
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    _path = path.join("app", "config.json")
+    with open(_path, "r") as file:
+        data_from_file = json.load(file)
+        fuel_price = data_from_file["FUEL_PRICE"]
+
+        customers_list = [
+            Customer.create_customer(customer)
+            for customer in data_from_file["customers"]
+        ]
+
+        shops_list = [
+            Shop.create_shop(shop) for shop in data_from_file["shops"]
+        ]
```

Comment:
The code assumes that the JSON file contains a list under the key 'shops'. Consider adding error handling to manage cases where this key might be missing or the data is not in the expected format.